src/main/java/uk/gov/hmcts/juror/api/moj/repository/CourtQueriesRepositoryImpl.java

```diff
@@ -58,6 +58,7 @@ public List<String> getAllCourtLocCodes() {
         return queryFactory
             .selectDistinct(COURT_LOCATION.locCode)
             .from(COURT_LOCATION)
+            .where(COURT_LOCATION.locCode.ne("400"))
```

Comment:
Would be worth updating the method name to indicate it excludes bureau or add excludes as a param

Reply:
updated to include a flag